### PR TITLE
Added group `cat`

### PR DESCRIPTION
**Pull request by RichardLitt**
This one is pretty straightforward.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -950,6 +950,170 @@ Outputs the list of peers that were removed.
 
 # Group cat
 
+Show IPFS object data
```
Comment (dignifiedquire) [inline apiary.apib]: Missing `.`

File: apiary.apib [hunk 2]
```diff
@@ -950,6 +950,170 @@ Outputs the list of peers that were removed.
...
+          "Message": "this dag node is a directory",
+          "Code": 0
+        }
+
++ Request With Valid Argument
+
+    In this request, 'QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o' is the hash for a file, 'test',
+    with the following contents
+
+        hello world
```
Comment (dignifiedquire) [inline apiary.apib]: Why not use backticks for consistency?

File: apiary.apib [hunk 3]
```diff
@@ -950,6 +950,170 @@ Outputs the list of peers that were removed.
...
+        ```
+        Content-Type: text/plain
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        X-Stream-Output: 1
+        Date: Fri, 05 Feb 2016 17:18:10 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (string)
```
Comment (dignifiedquire) [inline apiary.apib:2616]: Maybe add a description that this is the raw content of the file?

Comment (RichardLitt) [inline apiary.apib:2616]: Done. RFM?

File: apiary.apib (hunk 2 above)
Comment (RichardLitt) [inline apiary.apib]: Because no need if it is tabbed.

**Comment by RichardLitt** [comment]
@dignifiedquire final look through? RFM?

File: apiary.apib (hunk 2 above)
Comment (dignifiedquire) [inline apiary.apib]: I know but it's so inconsistent, as we use backticks everywhere else :(

Comment (RichardLitt) [inline apiary.apib]: Hahaha. But I just removed them because you told me to! :D

Comment (dignifiedquire) [inline apiary.apib]: lol

File: apiary.apib [hunk 4]
```diff
@@ -646,7 +646,9 @@ Stores input as an IPFS block
 
     In the example provided, the 'test' file has the following contents:
 
+        ```
         hello world
+        ```
```
Comment (dignifiedquire) [inline apiary.apib:651]: If you use backticks shouldn't the indentation be one level less?

Comment (RichardLitt) [inline apiary.apib:651]: You're killing me Friedel. 

Either we go with backticks at this level, or I change everything.

**Pull request merged**